usr/share/rear/prep/GNU/Linux/300_include_grub_tools.sh

```diff
@@ -3,6 +3,16 @@
 
 test -d $VAR_DIR/recovery || mkdir -p $VAR_DIR/recovery
 
+echo 'arch' $ARCH
```

Comment:
What is the purpose of the `echo` command?
Usually it would output to the ReaR log file, cf.
"What to do with stdin, stdout, and stderr" in
https://github.com/rear/rear/wiki/Coding-Style
but to get something only in the ReaR log file
we have the `Log()` function (which adds a timestamp),
see the various functions for messaging, logging and user input
in usr/share/rear/lib/_input-output-functions.sh

Reply:
it is removed. I had it in there while learning and forgot to remove